### ISSUE TITLE
Flapdoodle embed needs an update to work fully with Java 21

### ISSUE THREAD

**Issue by jkwatson-verta** [madansp/nats-embedded#11]
```
java.lang.NoSuchFieldException: handle
	at java.base/java.lang.Class.getDeclaredField(Class.java:2782)
	at de.flapdoodle.embed.process.runtime.Processes.windowsProcessId(Processes.java:112)
	at de.flapdoodle.embed.process.runtime.Processes.access$200(Processes.java:48)
	at de.flapdoodle.embed.process.runtime.Processes$PidHelper$3.getPid(Processes.java:218)
	at de.flapdoodle.embed.process.runtime.Processes.processId(Processes.java:81)
	at de.flapdoodle.embed.process.runtime.ProcessControl.<init>(ProcessControl.java:65)
	at de.flapdoodle.embed.process.runtime.ProcessControl.start(ProcessControl.java:165)
	at de.flapdoodle.embed.process.runtime.AbstractProcess.<init>(AbstractProcess.java:98)
	at np.com.madanpokharel.embed.nats.NatsServerProcess.<init>(NatsServerProcess.java:48)
	at np.com.madanpokharel.embed.nats.NatsServerExecutable.start(NatsServerExecutable.java:44)
	at np.com.madanpokharel.embed.nats.NatsServerExecutable.start(NatsServerExecutable.java:32)
	at de.flapdoodle.embed.process.runtime.Executable.start(Executable.java:109)
	at np.com.madanpokharel.embed.nats.EmbeddedNatsServer.startServer(EmbeddedNatsServer.java:53)
```

Looks like this was fixed in flapdoodle.embed, but requires an update to v 4: https://github.com/flapdoodle-oss/de.flapdoodle.embed.process/issues/137

**Comment by madansp** [comment]
This has been fixed in latest version 2.1.2

**Comment by jkwatson-verta** [comment]
Thanks!
